### PR TITLE
feat: don't run Tokenserver migrations on startup

### PR DESCRIPTION
**Pull request by ethowitz**
## Description

Since we're connecting to the pre-existing Tokenserver database, we should not be running the migrations on server startup.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -44,10 +44,13 @@ python:
 	venv/bin/python -m pip install -r requirements.txt
 
 run: python
-	PATH=./venv/bin:$(PATH) RUST_LOG=debug RUST_BACKTRACE=full cargo run -- --config config/local.toml
+	PATH="./venv/bin:$(PATH)" RUST_LOG=debug RUST_BACKTRACE=full cargo run -- --config config/local.toml
```
Comment (ethowitz) [inline Makefile:47]: This fixes an unrelated bug that causes `make run` to fail if an item in your path has any whitespace characters in it.

**Review by pjenvey — CHANGES_REQUESTED** [review]
This needs the e2e tests in CI to run them

**Comment by ethowitz** [comment]
Removed the `tokenserver_migrate` make task since the migrations will run automatically for any non-release build

**Comment by ethowitz** [comment]
The e2e tests run against a release build...I will add a line to run the migrations manually in the docker-compose

**Review by pjenvey — APPROVED** [review]

**Review by jrconlin — APPROVED** [review]

**Pull request merged**